man/sysusers.d.xml

```diff
@@ -100,15 +103,16 @@
   <refsect1>
     <title>Configuration File Format</title>
 
-    <para>The file format is one line per user or group containing
-    name, ID, GECOS field description and home directory:</para>
+    <para>The file format is one line per user or group containing name, ID, GECOS
+    field description, home directory, and login shell:</para>
 
-    <programlisting>#Type  Name   ID              GECOS                 Home directory
-u      httpd  440             "HTTP User"
-u      authd  /usr/bin/authd  "Authorization user"
-g      input  -               -
-m      authd  input
-u      root   0               "Superuser"           /root</programlisting>
+    <programlisting>#Type Name     ID             GECOS                 Home directory Shell
+u     httpd    440            "HTTP User"
```

Comment:
btw, if we wanted to be funny, we'd change the UID of this example line for "httpd" to "404" ;-)